### DISCUSSION TITLE
Web Interface and SSH

### DISCUSSION THREAD

**Discussion by robotgoat** [openmiko/openmiko#100] [Q&A]
Hello, forgive me if this feature already exists and I missed it. I followed the installation instructions to get the firmware on my Pan V1, which worked successfully. The camera connects to the wireless and I can see the hostname "openmiko" in the router dhcp leases. I can view the http stream and rtsp streams successfully as well. However, there does not seem to be a web interface or documentation on the default ssh user and password. Does the web interface exist and if the ssh username and password exist, what would they be? Thanks.

**Comment by robotgoat** [comment]
Update: I did miss the SSH info. It's in the FAQ. Perhaps this information can be placed in the install/setup section of the readme?

**Reply by numericOverflow** [reply]
@robotgoat - Did you ever figure out the web interface?

I'm in the same boat.  Migrating from Dafang-Hacks, and that had a pretty robust web-UI, but can't find anything here on OpenMiko so far.

I *can* do everything over SSH, but sometimes a nice webUI is so much quicker for simple stuff.

**Reply by robotgoat** [reply]
Unfortunately I haven't looked into it much. I ended up swapping out my Wyzecams for wired POE cameras and haven't looked into this too deeply.
